packages/trader/src/Modules/Trading/Components/Form/screen-large.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { TradeParamsLoader } from 'App/Components/Elements/ContentLoader';
 import Fieldset from 'App/Components/Form/fieldset.jsx';
-import ContractType from '../../Containers/contract-type.jsx';
+import ContractType from '../../Containers/contract-type';
```

Comment:
Can we remove the `.jsx` from the other imports while we're at it? Should the file names be CapitalCase too?

Reply:
which file are you referring to ?

Reply:
```suggestion
import ContractType from '../../Containers/ContractType';
```

Reply:
Never mind, seems it's the other files that are breaking [convention](https://wikijs.deriv.cloud/en/Frontend/development-guidelines/code-style#import-style)?

Reply:
😢

Reply:
file names at this moment are all `abc-def.ext` in the whole repo

Reply:
Another convention is that currently we leave `.jsx` for jsx files and no extension for tsx files.